lib/widgets/CustomButton/custom_button.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:classroom_app/styles/color/colors.dart';
+
+class CustomButton extends StatelessWidget {
+  const CustomButton({
+    super.key,
+    required this.buttonText,
+    required this.buttonBgColor,
+    this.buttonTextColor,
+    this.buttonBorderColor,
+  });
+
+  final String buttonText;
+  final Color buttonBgColor;
+  final Color? buttonTextColor;
+  final Color? buttonBorderColor;
+
+  @override
+  Widget build(BuildContext context) {
+    double screenWidth = MediaQuery.of(context).size.width;
+    return SizedBox(
+      height: 56,
+      width: screenWidth * 0.4,
+      child: ElevatedButton(
+        onPressed: () {},
+        style: ElevatedButton.styleFrom(
+          padding: EdgeInsets.symmetric(
+            horizontal: screenWidth * 0.08,
+            vertical: 14,
+          ),
+          backgroundColor: buttonBgColor,
+          side: buttonBorderColor == null
+              ? null
+              : BorderSide(color: buttonBorderColor!),
+          shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(5),
+          ),
```

Comment:
Default border radius (5) and font family (Poppins) might not align with app-wide design changes.
Use constants or theme values for these.
```suggestion
          shape: RoundedRectangleBorder(
            borderRadius: BorderRadius.circular(AppStyles.defaultBorderRadius),
          ),
```

Reply:
ditto

Reply:
done